src/tiledb/cloud/dag/dag.py

```diff
@@ -1083,12 +1078,12 @@ def submit_udf_stage(
             **kwargs,
         )
 
-    def submit_sql(self, *args: Any, **kwargs: Any) -> Node:
+    def submit_sql(self, sql: str, *args: Any, **kwargs: Any) -> Node:
```

Comment:
@ryan-williams If we pick `sql` out of `*args`, then I think we have to pass it explicitly on line 1092, yes?

Reply:
yikes, yes, good catch. Looks like it failed some tests as well, thankfully.

Reply:
I'm glad we have a test that covers this! https://github.com/TileDB-Inc/TileDB-Cloud-Py/actions/runs/11842972791/job/33002846291?pr=680#step:12:411